### ISSUE TITLE
Webui warns of expiring subscriptions when never had any (New behaviour in 2022.12)

### ISSUE THREAD

**Issue by digdilem** [uyuni-project/uyuni#6375]
### Problem description

Upgraded to 2022.12 yesterday. No errors reported during process.

Today, this warning appeared

![image](https://user-images.githubusercontent.com/4052275/208849508-68773c66-8929-4a26-a28d-dd502d23ce2c.png)

(Click)

![image](https://user-images.githubusercontent.com/4052275/208849613-63430979-a6bd-492b-bd8a-ae7ed42bad3b.png)

(Click)

![image](https://user-images.githubusercontent.com/4052275/208849668-8f177657-e2d6-4922-acef-caa4504a9ee4.png)

I don't think Uyuni should warn the user of subscriptions expiring when there are no subscriptions present. (Uyuni used here with Centos/Rocky vms)

Unclear whether this is a one-off or a recurring warning. 

### Steps to reproduce

As above

### Uyuni version

```bash
2022.12
```

**Comment by mbussolotto** [comment]
can you please run on your uyuni server:
```
echo "select * from susesccsubscription where subtype != 'internal' and (status = 'ACTIVE' and expires_at < now() + interval '90 day') or (status = 'EXPIRED' and expires_at > now() - interval '30 day')" | spacewalk-sql --select-mode -
```
it should show which subscription is triggering this warning. Thanks!

**Comment by mbussolotto** [comment]
CC @cbbayburt the result of the query that should trigger this warning seems ok...maybe something wrong on UI?

**Comment by santeri3700** [comment]
I have the same issue with an updated Uyuni 2022.12 server which has never been connected or subscribed to anything SCC related. \
Manually running the "gatherer-matcher-bunch" bunch did not resolve the issue.

There are some warning messages in the "Subscription Matching" page (not an issue to me):
- System has an unknown number of sockets, assuming 16
- Virtual guest has unknown host, assuming it is a physical system

PGSQL data:
```
echo "select * from susesccsubscription" | spacewalk-sql --select-mode -
 id | scc_id | credentials_id | name | starts_at | expires_at | status | regcode | subtype | system_limit | created | modified 
----+--------+----------------+------+-----------+------------+--------+---------+---------+--------------+---------+----------
(0 rows)
```

**Comment by gabjef** [comment]
I am seeing the same behavior with version 2022.11-220400.195.1.uyuni2.

"You have subscriptions that are expiring soon or already expired. Please check the Subscription Matching page."

Checking for expired subscriptions yields nothing:
` # echo "select * from susesccsubscription where subtype != 'internal' and (status = 'ACTIVE' and expires_at < now() + interval '90
day') or (status = 'EXPIRED' and expires_at > now() - interval '30 day')" | spacewalk-sql --select-mode -
 id | scc_id | credentials_id | name | starts_at | expires_at | status | regcode | subtype | system_limit | created | modified
----+--------+----------------+------+-----------+------------+--------+---------+---------+--------------+---------+----------
(0 rows)`

**Comment by santeri3700** [comment]
@gabjef you are probably actually running 2022.12, but the `Uyuni-Server-release` package version information is wrong (not sure if the repo is correct). \
I don't see this behavior on another actual 2022.**11** server so this bug was almost certainly introduced with 2022.**12**.

Fully up-to-date 2022.12 server (I also see wrong package version here):
```
zypper info Uyuni-Server-release
...
Repository     : uyuni-server-stable
Name           : Uyuni-Server-release
Version        : 2022.11-220400.195.1.uyuni2
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes
Status         : up-to-date
...

spacecmd -q -- api api.systemVersion
[
  "2022.12"
]
```

**Comment by digdilem** [comment]
Looks like this warning appears every day since 2022.12 upgrade, I had a new one this morning identical to the first. 

No other affects noted.

**Comment by saquib-akhtar** [comment]
This has come like a new feature where subscription expiry is synced from SUSE account attached with Uyuni

**Comment by mbussolotto** [comment]
CC @mseidl seems the issue was introduced by https://github.com/uyuni-project/uyuni/commit/3ed5b838f8c61a780c4f864c1c53c977c21f6cc2 ...can you please have a look ? Thanks !

**Comment by ppanon2022** [comment]
Similar problem with warnings despite a lack of subscriptions. There are 3 "Unmatched products": OpenSUSE 15, SUSE Manager Mgmt Single 1.2, and SUSE Manager Server 4.0 x86_64, even though these are all for the Uyuni server and 3 proxies all running on OpenSUSE. Perhaps the issue is the misclassification of the Uyuni Server and Proxy as SUSE Manager instances, which might require subscriptions?

**Comment by santeri3700** [comment]
The daily false subscription expiration notifications can be disabled by adding the following setting to the `/etc/rhn/rhn.conf` configuration. \
This does not hide the warning from the Web UI Overview, but stops emails and Web UI notification messages (under bell symbol).
```
java.notifications_type_disabled = ChannelSyncFinished,SubscriptionWarning
```

More information about the notifications can be found from the documentation: https://www.uyuni-project.org/uyuni-docs/en/uyuni/administration/troubleshooting/tshoot-notifications.html

List of all notification types are defined in here: https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/redhat/rhn/domain/notification/types/NotificationType.java

**Comment by ftigges-comdev** [comment]
Same here.
I do have an account at scc.suse.com. But no subscriptions.
If you need someone for testing, just send me the patch.

Best regards

```
uyuni:~ # echo "select * from susesccsubscription" | spacewalk-sql --select-mode -
 id | scc_id | credentials_id | name | starts_at | expires_at | status | regcode | subtype | system_limit | created | modified 
----+--------+----------------+------+-----------+------------+--------+---------+---------+--------------+---------+----------
(0 rows)

uyuni:~ # zypper info Uyuni-Server-release
Loading repository data...
Reading installed packages...


Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : Uyuni Server Stable
Name           : Uyuni-Server-release
Version        : 2022.11-220400.195.1.uyuni2
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : Uyuni-Server-release-2022.11-220400.195.1.uyuni2.src
Summary        : Uyuni Server
Description    : 
    Uyuni lets you efficiently manage physical, virtual,
    and cloud-based Linux systems. It provides automated and cost-effective
    configuration and software management, asset management, and system
    provisioning.
```

**Comment by juliogonzalez** [comment]
For reference, @mseidl is working on a fix that we expect to be part of 2023.01

**Comment by saquib-akhtar** [comment]
Now even when, the subscription got renewed last week in my INFRA for SUSE portal, then also continuous old BLANK Alerts are coming continuously on daily basis.

**Comment by mseidl** [comment]
@saquib-akhtar Do you mean blank emails?  Or a blank notification in uyuni?  (please attach a screenshot if you could)

**Comment by saquib-akhtar** [comment]
> @saquib-akhtar Do you mean blank emails? Or a blank notification in uyuni? (please attach a screenshot if you could)

Yes, I am attaching mail snapshots here. Both Blank Mails and WebUI warnings are getting continuously generated.

**Comment by saquib-akhtar** [comment]
![image](https://user-images.githubusercontent.com/103385358/213431960-4aee0790-2e2c-47c3-be88-6cb7bd3d88a2.png)

**Comment by mseidl** [comment]
Merged now.

**Comment by juliogonzalez** [comment]
To be released with 2023.01
